### DISCUSSION TITLE
Table without table markup

### DISCUSSION THREAD

**Discussion by olimart** [twbs/bootstrap#38005] [Q&A] [answered]
Anyone has an example of building table without `<table>` tag with Bootstrap 5?
There used to be an example in the docs (I think responsive tables were built without `table` tag in the past).

I would need alternate colour rows as well.

**Comment by tbolt** [comment, marked as answer]
I don't think Bootstrap has ever supported having tables without `<table>`

It wouldn't be advisable anyway given the reduced accessibility it would have.
